src/components/sbb-checkbox-group/sbb-checkbox-group.scss

```diff
@@ -0,0 +1,46 @@
+// Default component properties, defined for :host. Properties which can not
```

Comment:
There is an unwanted space probably:
<img width="462" alt="Bildschirmfoto 2022-11-15 um 16 58 52" src="https://user-images.githubusercontent.com/29278232/201966703-f215c5c7-dc0f-4fee-a0a7-a89c875a217c.png">